app/controllers/spree/admin/gift_cards_controller.rb

```diff
@@ -1,17 +1,19 @@
 class Spree::Admin::GiftCardsController < Spree::Admin::BaseController
-  before_filter :load_gift_card_history, only: [:show]
   before_filter :load_user, only: [:lookup, :redeem]
   before_filter :load_gift_card_for_redemption, only: [:redeem]
   before_filter :load_gift_card_by_id, only: [:edit, :update, :send_email]
   before_filter :load_order, only: [:edit, :update]
+  Spree::LineItem.whitelisted_ransackable_associations += %w[order]
```

Comment:
I'm not sure what the best way to do this actually is. I need the order association on the line item to be able to search by order number. This is not a good way to do it, so looking for a recommended better way.